test/GameState.test.js

```diff
@@ -0,0 +1,58 @@
+const expect = require("chai").expect;
+const { GameState } = require("../entities/GameState");
+const { createPlayer } = require("./testutils");
+
+describe("GameState", () => {
+  let gameState;
+  beforeEach(() => {
+    gameState = new GameState();
+  });
+  it("should load default values", () => {
+    expect(gameState.turn).equal(0);
+    expect(gameState.turnTaken).to.be.false;
+    expect(gameState.players.length).equal(0);
+    expect(gameState.allPlayerActions).to.be.an("object");
+    expect(gameState.allPlayerActions).not.to.include.keys("ROLL_DICE");
+    expect(gameState.currentPlayerActions).to.be.an("object");
+    expect(gameState.currentPlayerActions).not.to.include.keys("ROLL_DICE");
+  });
+  it("should not throw error if no players and trying to get current player", () => {
```

Comment:
Can we also add the opposite case wherein we test that it does throw given some circumstance (I guess as a contra to the `it` description ?)

Reply:
it will never throw errors. I'll rename the test